inc/template-tags.php

```diff
@@ -562,13 +563,30 @@ function responsive_get_posts_archive_link() {
 		'meta_value' => 'page-templates/news.php',
 	) );
 
-	// Find the first news page set to display "All Categories".
+	$_post = get_post();
+	$post_cats = get_the_terms( $_post, 'category' );
```

Comment:
Rather than defining a `$_post` variable that is only used once, maybe you could change to just be
```
$post_cats = get_the_terms( get_post(), 'category' );
// OR
$post_cats = get_the_terms( get_queried_object_id(), 'category' );
// OR:
$post_cats = get_the_terms( get_the_ID(), 'category' );
```

I tend to avoid get_post calls if I just need the ID, but I bet it doesn't matter so much with caching and all that -- those methods probably can easily retrieve the already queried post object.